Library/Homebrew/test/cask/dsl_spec.rb

```diff
@@ -168,37 +168,37 @@
         cask.config = config
       end
 
-      context "to 'zh'" do
+      context "when it is 'zh'" do
```

Comment:
The wording for these feel a tad off:

```
Cask::DSL language stanza when language is set explicitly when it is 'zh'
```

I almost wonder if the `it` should be allowed for these cases. Either way, it's clear in what it means so not a blocker.

Reply:
Maybe `Cask::DSL language stanza when language is set explicitly with 'zh'` as a _slightly_ more understandable alternative? Or we can add "to" to the allowed starters (https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/ContextWording)?